user/serializers.py

```diff
@@ -1,13 +1,57 @@
-from django.contrib.auth import get_user_model
+from django.contrib.auth import get_user_model, authenticate
 from rest_framework import serializers
+from django.utils.translation import gettext as _
+
+
+class AuthTokenSerializer(serializers.Serializer):
+    email = serializers.CharField(label=("email",), write_only=True)
+    password = serializers.CharField(
+        label=("Password",),
+        style={
+            "input_type": "password",
+        },
+        trim_whitespace=False,
+        write_only=True,
+    )
+    token = serializers.CharField(label=("Token",), read_only=True)
+
+    def validate(self, attrs):
+        email = attrs.get("email")
+        password = attrs.get("password")
+
+        if email and password:
+            user = authenticate(
+                request=self.context.get("request"),
+                email=email,
+                password=password
+            )
+
+            # The authenticate call simply returns None for is_active=False
+            # users. (Assuming the default ModelBackend authentication
+            # backend.)
+            if not user:
+                msg = _("Unable to log in with provided credentials.")
+                raise serializers.ValidationError(msg, code="authorization")
+        else:
+            msg = _('Must include "username" and "password".')
```

Comment:
The error message mentions 'username' instead of 'email'. Update the message to reflect the correct field: 'Must include "email" and "password".'